Packages.props

```diff
@@ -10,7 +10,7 @@
     <PackageReference Update="Prism.Unity.Forms" Version="$(PrismVersion)" />
     <PackageReference Update="Prism.DryIoc.Forms" Version="$(PrismVersion)" />
     <PackageReference Update="Prism.Plugin.Popups" Version="7.2.0.1046" />
-    <PackageReference Update="Prism.Plugin.Logging.Loggly" Version="7.2.0.1114" />
+    <PackageReference Update="Prism.Plugin.Logging.Loggly" Version="8.0.11-beta" />
```

Comment:
remove this

Reply:
Removing this change will cause advanced-topics/Logging-Loggly to no longer be able to build.
It was failing due to a update to Prism 8. The version of Prism.Plugin.Logging.Loggly that was referenced expected an ILoggerFacade.

Reply:
oh, sorry, I should have paid more attention. This is not the contoso sample. I though all changes were made to a single  sample. Disregard